src/styles/blocks/card.scss

```diff
@@ -0,0 +1,121 @@
+.card {
+  background-color: $white-color;
+  border: 1px solid $gray-elements-color;
+  border-radius: 5px;
+  padding: 32px 0 16px;
+  padding-inline: 16px;
+  width: $card-width;
+
+  transition:
+    transform 300ms,
+    color 300ms,
+    background 300ms;
+
+  &:hover {
+    transform: scale(110%);
```

Comment:
The hover effect for scaling is set to 110%, which may still be too large and could disrupt the layout. Please adjust this to a smaller scale factor to maintain visual appeal and ensure it doesn't affect the surrounding layout.